test/cpp/jit/test_misc.cpp

```diff
@@ -1461,6 +1461,18 @@ TEST(TestSymInt, TestIntrusive) {
   ASSERT_EQ(b.use_count(), 3);
 }
 
+TEST(TestSymInt, TestIntrusiveScalar) {
+  auto a = c10::make_intrusive<c10::SymIntNodeImpl>();
```

Comment:
There seems to be a few different scenarios we might want to test
Axis 1 : copying from Scalar(int) into Scalar(SymInt) and vice versa
Axis 2: different types of copying: copy c-tor, assignment operator w/ , w/o move